docs/maps/maps-getting-started.asciidoc

```diff
@@ -62,14 +61,15 @@ and lighter shades will symbolize countries with less traffic.
 
 . Add a Tooltip field:
 
-** Select **ISO 3166-1 alpha-2 code** and **name**.
-** Click **Add**.
+** **ISO 3166-1 alpha-2 code** is added by default.
+** Click **Add** and select **name**.
```

Comment:
How about changing to the below. Not sure "Click **Add** to create the field" makes sense since you are not "creating a field"
```
** Click **+ Add** to open field select.
** Select **name** and click *Add*.
```